components/molecules/InsightRow/insight-row.tsx

```diff
@@ -31,18 +33,30 @@ const InsightRow = ({ insight, user }: InsightRowProps) => {
           <div className="flex items-center lg:items-center gap-4 ">
             <div className="w-4 h-4 bg-light-orange-10 rounded-full"></div>
             <div className="text-xl text-light-slate-12 flex justify-between">
-              <Link href={`/pages/${user?.user_metadata.user_name}/${insight.id}/dashboard`}>{insight.name}</Link>
-            </div>
-            <div className="rounded-2xl border px-2 text-light-slate-11">
-              {!!insight.is_public ? "public" : "private"}
+              <Link href={`/pages/${user ? user?.user_metadata.user_name : "anonymous"}/${insight.id}/dashboard`}>
+                {!isEditable ? `Demo | ${insight.name}` : insight.name}
+              </Link>
             </div>
-            <div className="flex-1 md:hidden">
-              <span className=" bg-light-slate-1 inline-block rounded-lg p-2.5 border mr-2">
-                <Link href={`/hub/insights/${insight.id}/edit`}>
-                  <BsPencilFill title="Edit Insight Page" className="text-light-slate-10 text-md cursor-pointer w-4" />
-                </Link>
-              </span>
+            <div
+              className={clsx(
+                "rounded-2xl border px-2 text-light-slate-11",
+                !isEditable ? "text-sauced-orange bg-orange-200 border-orange-500" : ""
```

Comment:
a11y (blocking): The colour contrast is failing for all levels. @isabensusan, can we get this tweaked so it passes colour contrast checks?

![CleanShot 2023-12-04 at 19 43 00](https://github.com/open-sauced/app/assets/833231/e3fcb7bf-a6c8-4b3a-aadc-c7820cf6a49b)

Reply:
let's use
bg-orange-50 for the background
text-orange-700 for the text
border-orange-600 for the border

It passes the minimum contrast requirements and only fails the enhanced standard, which seems to be the case with most pill components I've encountered in big platform's design systems, so it's good enough to ship.

<img width="478" alt="image" src="https://github.com/open-sauced/app/assets/27322879/c0fc75f2-c70a-48c0-93df-d2a791395602">